pkg/models/html.go

```diff
@@ -0,0 +1,78 @@
+package models
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"html/template"
+	"io"
+
+	"github.com/cmsgov/easi-app/pkg/sanitization"
+)
+
+// HTML represents html code. It is sanitized to only allow specific tags
```

Comment:
I think this could comment could be a bit clearer, to clarify whether an `HTML` value should always be sanitized when it's constructed, or whether it needs to be sanitized before usage. (It looks like the former is what's intended) It's not a big issue, though.